Source/WebCore/Modules/model-element/HTMLModelElement.cpp

```diff
@@ -377,6 +388,37 @@ void HTMLModelElement::applyBackgroundColor(Color color)
         m_modelPlayer->setBackgroundColor(color);
 }
 
+#if ENABLE(MODEL_PROCESS)
+const DOMMatrixReadOnly& HTMLModelElement::entityTransform() const
+{
+    return m_entityTransform.get();
+}
+
+ExceptionOr<const DOMMatrixReadOnly&> HTMLModelElement::setEntityTransform(const DOMMatrixReadOnly& transform)
```

Comment:
This should be `ExceptionOr<void>`. A setter doesn't return anything. You'll also need to adjust wherever you attempt to return things below that are not exceptions.

Reply:
Got it. Fixed! Thank you.